src/Callback.php

```diff
@@ -132,6 +132,27 @@ public function set($callback, $args = [])
         }
     }
 
+    /**
+     * Terminate this callback
+     * by rendering the owner view.
+     */
+    public function terminate()
+    {
+        if ($this->canTerminate()) {
+            $this->app->terminateJSON($this->owner);
+        }
+    }
+
+    /**
+     * Prevent callback from terminating during a reload.
+     */
+    protected function canTerminate(): bool
+    {
+        $reload = $_GET['__atk_reload'] ?? null;
```

Comment:
Seems like a hack to me which will probably not work if nested multiple times. Few more hacks and the code will explode.

Reply:
@mvorisek - chill out man - all lights are green - nothing will explode 😄 
Besides, this fixes a real problem - if you have something better to propose please do.